extensions/kubectl-cli/package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "kubectl-cli-tool",
```

I guess we can shorter the name as well
```suggestion
  "name": "kubectl-cli",
```